### DISCUSSION TITLE
domain hosting

### DISCUSSION THREAD

**Discussion by arshanashok70** [community/community#39690] [GitHub Education] [answered]
is there any domain hosting services available in student developer pack .

**Comment by david-engelmann** [comment]
There are a few options, [this site has the components of the student developer pack](https://education.github.com/pack). You will see Namecheap, Name.com, .TECH, as well as DigitalOcean for cloud hosting.

**Reply by arshanashok70** [reply]
the first 3 only provides free domains only , and digital ocean have some issues while opening , is there any other services available for domain hosting.

**Comment by david-engelmann** [comment]
You can use the free domains from one of the sites in the student developer pack such as Name.com and host it using Github Pages. [Check out this article that walks through the process step by step](https://medium.com/@jai.dewani.99/hosting-your-github-web-page-on-custom-domain-name-freely-using-github-student-pack-80d00a343b90)!

**Comment by arshanashok70** [comment]
How to get CSR code from GitHub for domain which I hosted in GitHub pages. Sent from Mail for Windows From: David EngelmannSent: Tuesday, November 22, 2022 1:09 AMTo: community/communityCc: arshanashok70; AuthorSubject: Re: [community/community] domain hosting (Discussion #39690) You can use the free domains from one of the sites in the student developer pack such as Name.com and host it using Github Pages. Check out this article that walks through the process step by step!—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by david-engelmann** [comment, marked as answer]
[There is a great answer to the CSR code question here](https://stackoverflow.com/questions/65377212/how-to-get-a-csr-for-github-pages-website)
